pennylane/qinfo/entropies.py

```diff
@@ -53,3 +53,53 @@ def wrapper(*args, **kwargs):
         return entropy
 
     return wrapper
+
+
+def mutual_info_transform(qnode, indices0, indices1, base=None):
+    r"""Compute the mutual information from a :class:`.QNode` returning a :func:`~.state`:
+
+    .. math::
+        I(A, B) = S(\rho^A) + S(\rho^B) - S(\rho^{AB})
```

Comment:
I cannot find why but in readsthedocs it is not showing right.

Reply:
Fixed.